data/scripts/weapons/#example.lua

```diff
@@ -0,0 +1,83 @@
+--[[
+		
+	Burst Arrow example
+
+]]
+local area = createCombatArea({
+	{1, 1, 1},
+	{1, 3, 1},
+	{1, 1, 1}
+})
+
+local combat = Combat()
+combat:setParameter(COMBAT_PARAM_TYPE, COMBAT_PHYSICALDAMAGE)
+combat:setParameter(COMBAT_PARAM_EFFECT, CONST_ME_EXPLOSIONAREA)
+combat:setParameter(COMBAT_PARAM_BLOCKARMOR, true)
+combat:setFormula(COMBAT_FORMULA_SKILL, 0, 0, 1, 0)
+combat:setArea(area)
+
+local burstarrow = Weapon(WEAPON_AMMO)
+
+burstarrow.onUseWeapon = function(player, variant)
+	if player:getSkull() == SKULL_BLACK then
+		return false
+	end
+
+	return combat:execute(player, variant)
+end
+
+burstarrow:id(2546)
+burstarrow:attack(27)
+burstarrow:shootType(CONST_ANI_BURSTARROW)
+burstarrow:ammoType("arrow")
+burstarrow:maxHitChance(100)
+burstarrow:register()
+
+--[[
+		
+	Wand of Vortex example
+
+]]
+local wov = Weapon(WEAPON_WAND)
+wov:id(2190)
+wov:minDamage(8)
+wov:maxDamage(18)
+wov:element("energy")
+wov:level(7)
+wov:mana(2)
+wov:vocation("sorcerer", true, true)
+wov:vocation("master sorcerer")
+wov:register()
+
+--[[
+		
+	Arbalest example
+
+]]
+local arbalest = Weapon(WEAPON_DISTANCE)
+arbalest:id(5803)
+arbalest:slotType("two-handed") -- it's now a 2h weapon
+arbalest:ammoType("bolt")
+arbalest:range(6)
+arbalest:attack(2)
+arbalest:hitChance(2)
```

Comment:
The problem is after looking a while at the code I noticed that this does not work the way you think right now it does, because I thought exactly the same.
If you put on a distance Weapon a `hitChance(2)` it means it'll give 2% more chance onto the `ammoType's` hit chance, if you put as an example `hitChance(100)` it means that with this weapon you'll 100% of times hit a target doesn't matter how much `maxHitChance` your ammo has.
While `maxHitChance` just indicates what's ranging from `0 up to maxHitChance` so if put `maxHitChance(100)` it'll still not hit 100% of the time, which is weird I know.